src/components/Menu/Menu/Menu.module.scss

```diff
@@ -3,6 +3,7 @@
 .menu {
   margin: unset;
   padding: unset;
+  position: relative;
```

Comment:
It should be on the MenuItem, no?

Reply:
Actually I see that it messes it up, maybe because there's some positioning already in it